examples/src/main/scala/org/apache/spark/examples/mllib/NaiveBayesExample.scala

```diff
@@ -15,6 +15,16 @@
  * limitations under the License.
  */
 
+/*
+ * Usage:
```

Comment:
So this usage text probably isn't the format we want, its pretty specific to each Spark release. Maybe look at some of the other files in this directory and see how they do their usage?